Makefile

```diff
@@ -600,3 +600,21 @@ kind-cluster: ## Create a new kind cluster designed for development with Tilt
 
 go-version: ## Print the go version we use to compile our binaries and images
 	@echo $(GO_VERSION)
+
+## --------------------------------------
+## Documentation and Publishing
+## --------------------------------------
+
+.PHONY: serve
+serve: ## Build the CAPIBM book and serve it locally to validate changes in documentation.
+	make -C docs/book/ serve
+
+## --------------------------------------
+## Update Go Version
+## --------------------------------------
+
+.PHONY: update-go
+update-go: ## Update Go version across files - Usage make update-version VERSION=X.YY.ZZ
```

Comment:
Really considering `make update-version X.Y.Z` than `make update-version VERSION=X.YY.ZZ`.
Will look into it.

Reply:
@kishen-v any update?